### ISSUE TITLE
Allow parsing from a Stream

### ISSUE THREAD

**Issue by bernardo-martinez** [AlexKovalevych/exoffice#24]
I have a case where I receive the xls|xlsx|csv file as a Base64 encoded String, so I need to create a file in order to be able to parse it using ExOffice. 

I think it would be nice to directly read from a Stream the file to parse instead of requiring a file path, i think it can be either by implementing it within existing `parse` and dispatching functionality depending on the type of the arg, or implement a new `parse_from_stream` function. WDYT?

 I can work on an implementation if you see it feasible and useful

**Comment by AlexKovalevych** [comment]
yeah, it would be nice to have, probably a separate `parse_stream` function
